### PR TITLE
add hooli-style-popup homework

### PR DESCRIPTION
**Pull request by bmukha**
### hooli-style-popup

[Demo](https://bmukha.github.io/hooli-style-popup/) |
[Code base](https://github.com/bmukha/hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/bmukha/hooli-style-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  <body>
+    <header class="header">
+      <div class="logo" tabindex="1">
+        <a href="https://kottans.org/" target="_blank"
+          ><img src="./popup-icons/favicon.png" width="32" alt="kottans logo"
+        /></a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__nav__items">
+          <li class="header__nav__item__main" tabindex="2">Menu</li>
```
Comment (HannaSyn) [inline submissions/bmukha/hooli-style-popup/index.html:21]: Logically, items in navigation should be links, so please add `a` inside `li`

File: submissions/bmukha/hooli-style-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                <input
+                  type="checkbox"
+                  name="more_checkbox"
+                  class="more__checkbox"
+                  id="more__checkbox"
+                />
+                <label
+                  for="more__checkbox"
+                  class="more__label"
+                  tabindex="5"
```
Comment (HannaSyn) [inline submissions/bmukha/hooli-style-popup/index.html:111]: `tabindex` is not necessary in this task, and in your case, it cause problems - when you navigate with tab, focus should appear on icons first, and then on `More` button. 
Also when focus on `More` , hidden menu is not opens by clicking space on keyboard.

File: submissions/bmukha/hooli-style-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        <a href="https://kottans.org/" target="_blank"
+          ><img src="./popup-icons/favicon.png" width="32" alt="kottans logo"
+        /></a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__nav__items">
+          <li class="header__nav__item__main" tabindex="2">Menu</li>
+          <li class="header__nav__item__main" tabindex="3">Items</li>
+          <li class="header__nav__item__main menu">
+            <label for="menu__toggle">
```
Comment (HannaSyn) [inline submissions/bmukha/hooli-style-popup/index.html:24]: Please, add `:focus` state to this element

File: submissions/bmukha/hooli-style-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+                  tabindex="5"
+                ></label>
+                <nav class="menu__wrapper__secondary">
+                  <ul class="menu__items__secondary">
+                    <li class="header__nav__item__secondary">
+                      <a href="#"
+                        ><img
+                          src="./popup-icons/apps.png"
+                          width="48"
+                          alt="apps"
```
Comment (HannaSyn) [inline submissions/bmukha/hooli-style-popup/index.html:120]: For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

File: submissions/bmukha/hooli-style-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  height: 100%;
+}
+.header__nav__items {
+  list-style-type: none;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
```
Comment (HannaSyn) [inline submissions/bmukha/hooli-style-popup/style.css:39]: Please, add empty line after each block

File: submissions/bmukha/hooli-style-popup/style.css [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+.menu__wrapper__secondary {
+  display: none;
+}
+
+.more__checkbox:checked ~ .menu__wrapper__secondary {
+  display: grid;
+}
+
+input[type="checkbox"]:checked + label::after {
```
Comment (HannaSyn) [inline submissions/bmukha/hooli-style-popup/style.css:83]: Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

**Review by HannaSyn — CHANGES_REQUESTED** [review]
Good work! 
See my comments below

**Comment by bmukha** [comment]
Tnx for the feedback. Provided links have been very insightful. Please, review my submission again - I believe all aforementioned issues have been addressed.

**Pull request merged**